terra-jupyter-aou/Dockerfile

```diff
@@ -85,9 +85,7 @@ ENV USER jupyter-user
 USER $USER
 
 RUN pip3 install --upgrade \
-  pandas-profiling==2.10.1 \
-  plotnine==0.7.1 \
-  # Parent image pins tensorflow to an old alpha version. Override here for now.
-  tensorflow==2.3.0 \
-  numpy==1.18.5 \
-  "git+git://github.com/all-of-us/workbench-snippets.git#egg=terra_widgets&subdirectory=py"
+  nbstripout \
+  "git+git://github.com/all-of-us/workbench-snippets.git#egg=terra_widgets&subdirectory=py" \
+  && mkdir -p /home/$USER/.config/git \
```

Comment:
Curious if we'd want what's in this directory to persist on the user's persistent disk? if so, everything under the directory `/home/$USER/notebooks` persists since that's where we mount the PD

Reply:
For AoU, we want this to persist on the base image so that its enabled globally by default for all users. It keeps people from accidentally checking into source control notebooks *containing outputs*.